src/django_components/component.py

```diff
@@ -96,15 +95,14 @@ class RenderInput(Generic[ArgsType, KwargsType, SlotsType]):
     args: ArgsType
     kwargs: KwargsType
     slots: SlotsType
-    escape_slots_content: bool
```

Comment:
`escape_slots_content` can be passed to `Component.render()`, so that the results from user-supplied slots will be sanitised if not SafeText.

The way it is processed, though, is that if `escape_slots_content` is True, we pre-process the slot functions such that their output will be serialized.

So keeping the record of what value of `escape_slots_content` was passed to the component IMO doesn't provide any value. I can't imagine how I would use it. That's why I removed it.